src/main/java/seedu/academyassist/storage/JsonAdaptedPerson.java

```diff
@@ -142,7 +142,10 @@ public Person toModelType() throws IllegalValueException {
             }
             personSubjects.add(subject.toModelType());
         }
-        final Set<Subject> modelSubjects = new HashSet<>(personSubjects);
```

Comment:
Since `personSubjects` is no longer used for creating `modelSubjects`, would it make sense to remove lines 138-144?